jenkins.sh

```diff
@@ -28,3 +28,45 @@
 ./gradlew unitTest integrationTest \
     --profile --no-daemon --continue -PtestLoggingEvents=started,passed,skipped,failed "$@" \
     || { echo 'Test steps failed'; exit 1; }
+
+# Verify that Kafka Streams archetype compiles
+if [ $JAVA_HOME = "/home/jenkins/tools/java/latest11" ] ; then
```

Comment:
The created archetype, only works for Java8 -- we would need to create a different archetype for Java11.

What we could do, is to include the Java11 setting as comments in the archetype (thoughts)?

Reply:
We could do that, or we can just publish another archetype for use with Java 11+. Then, when we drop support for Java 8, it's as simple as just deleting the Java 8 archetype.

Reply:
Not sure how we could have two archetypes at the same time? I guess we would need to encode the Java version in the archetype name?
```
-DarchetypeGroupId=org.apache.kafka
-DarchetypeArtifactId=streams-quickstart-java   <--- change this to `streams-quickstart-java-8` and `streams-quickstart-java-11` ?
-DarchetypeVersion=$version 
```

Reply:
Yep, that would be the way to do it.

Reply:
@guozhangwang Can/should we do this in this PR? Or would this need a KIP? Or just create a JIRA for a follow up PR?

Reply:
We can create a JIRA ticket for a follow-up PR (this would need a KIP, yes).

Reply:
Created https://issues.apache.org/jira/browse/KAFKA-8116